helios-client/src/main/java/com/spotify/helios/common/descriptors/Descriptor.java

```diff
@@ -49,4 +49,9 @@ public String toJsonString() {
       throws IOException {
     return parse(value.getBytes(UTF_8), clazz);
   }
+
+  @Override
+  public String toString() {
+    return "Descriptor{}";
+  }
```

Comment:
since this is an abstract class, I don't think there is any value in providing this default implementation.

For instance if someone adds a subclass and forgets to override `toString()`, it would make more sense for `SomeNewClass.toString()` to return `com.spotify.helios.foo.SomeNewClass@123456` or whatever the default is rather than `Descriptior{}`.